### PR TITLE
some kr translation of raidboss (e1n~e4n)

### PR DESCRIPTION
**Pull request by haka5e**
first time using github. sorry for any problems.

### CODE REVIEW

**Comment by Jaehyuk-Lee** [comment]
하데스 토벌전 한국 지역 이름이 안들어가있네요

**Comment by quisquous** [comment]
No worries about problems! Thanks for the pull request, especially if this is your first time using github! More Korean cactbot translations are very welcome.

It looks like there's some errors here that the build caught: https://travis-ci.com/quisquous/cactbot/jobs/270433955.  You can test this locally by running `npm run lint` and `npm run test` (and `npm runlintfix` will fix some of these).  It looks like there's some tabs instead of spaces as well as some incorrect keys from copying and pasting.

If you can fix those up, commit them, and push that commit, I can merge this.  :+1:

File: ui/raidboss/data/05-shb/raid/e2n.js [hunk 1]
```diff
@@ -89,11 +98,13 @@
       regexDe: / 14:3E46:Arm Des Erebos starts using Gähnender Abgrund/,
       regexFr: / 14:3E46:Bras D'érèbe starts using Vaine Malice/,
       regexJa: / 14:3E46:エレボスの巨腕 starts using 虚ろなる悪意/,
+      regexJa: / 14:3E46:에레보스의 팔 starts using 공허한 악의/,
```
Comment (KattTails) [inline ui/raidboss/data/05-shb/raid/e2n.js]: This should be `      regexKo: / 14:3E46:에레보스의 팔 starts using 공허한 악의/,`

File: ui/raidboss/data/05-shb/raid/e2n.js [hunk 2]
```diff
@@ -103,6 +114,7 @@
       regexDe: / 14:3E42:Nichtswandler starts using Dunkel-Feuga/,
       regexFr: / 14:3E42:Marcheuse Du Néant starts using Méga Feu Ténébreux/,
       regexJa: / 14:3E42:ヴォイドウォーカー starts using ダークファイガ/,
+      regexJa: / 14:3E42:보이드워커 starts using 다크 파이가/,
```
Comment (KattTails) [inline ui/raidboss/data/05-shb/raid/e2n.js]: This should be `      regexKo: / 14:3E42:보이드워커 starts using 다크 파이가/,`

File: ui/raidboss/data/05-shb/raid/e2n.js [hunk 3]
```diff
@@ -131,13 +144,14 @@
...
             cn: '集合',
+            ko: '집합 -> YOU',
           };
         }
         return {
           en: 'Stack on ' + data.ShortName(matches[1]),
           de: 'Auf ' + data.ShortName(matches[1]) + ' sammeln',
           fr: 'Package sur '+ data.ShortName(matches[1]),
-          cn: '集合 -> ' + data.ShortName(matches[1]),
+          cn: '집합 -> ' + data.ShortName(matches[1]),
```
Comment (KattTails) [inline ui/raidboss/data/05-shb/raid/e2n.js]: You've replaced the cn entry here? It probably should be 
```
          cn: '集合 -> ' + data.ShortName(matches[1]),
          ko: '집합 -> ' + data.ShortName(matches[1]),
```

File: ui/raidboss/data/05-shb/raid/e2n.js [hunk 4]
```diff
@@ -172,6 +187,7 @@
         de: 'Verzögertes Feuer',
         fr: 'Feu retardé',
         cn: '延迟火',
+        cn: '지연술:파이어',
```
Comment (KattTails) [inline ui/raidboss/data/05-shb/raid/e2n.js]: This should be `       ko: '지연술:파이어',`

File: ui/raidboss/data/05-shb/raid/e1n.js [hunk 5]
```diff
@@ -333,5 +350,50 @@
...
+        '--untargetable--': '--타겟불가능--',
+        '--targetable--': '--타겟가능--',
+      },
+      '~effectNames': {
+        'Summon Order': 'Summon Order',
+        'Heavy': 'Heavy',
+        'Fetters': 'Fetters',
+        'Brink of Death': 'Brink of Death',
+      },
+	  },
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/data/05-shb/raid/e1n.js]: 맨 앞에 탭 지우시면 될 것 같아요

File: ui/raidboss/data/05-shb/raid/e1n.js [hunk 6]
```diff
@@ -333,5 +350,50 @@
...
+    {
+      'locale': 'ko',
+      'replaceSync': {
+        'Eden Prime': '에덴 프라임',
+        'Eden': '에덴',
+        'Guardian of Paradise': '에덴의 수호자',
+        'Engage!': '전투 시작!',
+      },
+      'replaceText': {
+        'attack': '공격',
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/data/05-shb/raid/e1n.js]: 맨 뒤에 스페이스 지우시면 될 것 같아요.

File: resources/lang/ko.js [hunk 7]
```diff
@@ -21,47 +21,47 @@ class CactbotLanguageKo extends CactbotLanguage {
...
-      Ewer: 'FIXME', // 0x75e
-      Spear: 'FIXME', // 0x75d
-      Spire: 'FIXME', // 0x75f
-      LadyOfCrowns: 'FIXME', // 0x755
-      LordOfCrowns: 'FIXME', // 0x754
+      Bole: '세계수의 줄기', // 0x75b
+      Ewer: '살리아크의 물병', // 0x75e
+      Spear: '할로네의 창', // 0x75d
+      Spire: '비레고의 탑', // 0x75f
+      LadyOfCrowns: '점지: 여왕의 날개', // 0x755
```
Comment (quisquous) [inline resources/lang/ko.js]: It looks like this is different from what was added in #887.  Which of these is the correct version?

**Comment by quisquous** [comment]
It looks like this is fine other than a few tab/spacing errors in e1n.js.  Can you fix those so that I can merge this? https://travis-ci.com/quisquous/cactbot/jobs/270610744

**Comment by quisquous** [comment]
I fixed up the merge conflicts and lint errors, so I could get this merged.  Thanks for the pull request!

**Pull request merged**